fail2ban/tests/clientreadertestcase.py

```diff
@@ -649,8 +649,9 @@ def testReadStockJailConfForceEnabled(self):
 
 			# and we know even some of them by heart
 			for j in ['sshd', 'recidive']:
-				# by default we have 'auto' backend ATM
-				self.assertTrue(['add', j, 'auto'] in comm_commands)
+				# by default we have 'auto' backend ATM, but some distributions can overwrite it, 
+				# (e.g. fedora default is 'systemd') therefore let check it without backend...
+				self.assertTrue([True for i in comm_commands if len(i) == 3 and i[0:2] == ['add', j]])
```

Comment:
If you'll take a look on #1492, you'll don't ask :) some things are better implementable over 0.10

I'll cherry pick and re-merge 0.9 to 0.10 hereafter (to create a clean merge point backwards, to resolve possible conflicts)...
I swear!